src/main/java/spoon/support/compiler/ZipFolder.java

```diff
@@ -39,6 +39,9 @@ public ZipFolder(File file) throws IOException {
 		if (!file.isFile()) {
 			throw new IOException(file.getName() + " is not a valid zip file");
 		}
+		if (false==true) {
+			String s = new String[0].toString();
+		}
```

Comment:
Guessing this is just to trigger an error?

Reply:
Yes